pyaedt/edb.py

```diff
@@ -3356,3 +3358,147 @@ def calculate_initial_extent(self, expansion_factor):
         max_width = max_width * expansion_factor
         self.logger.info("The W factor is {}, The initial extent = {:e}".format(expansion_factor, max_width))
         return max_width
+
+    @pyaedt_function_handler()
+    def copy_zones(self, working_directory=None):
+        """Copy multizone EDB project to one new edb per zone.
```

Comment:
```suggestion
        """Copy the multizone EDB project to one new edb per zone.
```
I see you didn't like "edb" capitalized in the latter half of the sentence. I'd like to learn why--in case I'm doing something wrong when making suggestions!